test/test_catkin_bundle.py

```diff
@@ -38,9 +37,11 @@ async def test_bundle():
     # Concise read on why it's patched this way.
     # http://www.voidspace.org.uk/python/mock/patch.html#where-to-patch
     with patch('colcon_ros_bundle.task.ros_bundle.RosdepWrapper') as wrapper:  # noqa: E501
-        wrapper().get_rule.side_effect = _get_rule_side_effect
-        wrapper().resolve.side_effect = _resolve_side_effect
-        await task.bundle()
+        with patch('os.environ') as environ:
```

Comment:
monkeypatch is what you want : https://docs.pytest.org/en/stable/reference.html#std-fixture-monkeypatch